plugins/generator-1.20.1/forge-1.20.1/templates/json/item_model_blocking.json.ftl

```diff
@@ -0,0 +1,49 @@
+{

```

Comment:
Given you close this { in each if, it would be more readable code if you opened { within the if statements too as moving this outside if will save ~two characters and cost readability

Reply:
✔️ Fixed